ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -1044,6 +1044,33 @@ then
 	fi
 
 	#
+	# RC car setup
+	#
+	if [ $VEHICLE_TYPE == rccar ]
```

Comment:
A couple of lines above there's also rover, i preferred to add it like this as I was not sure about the implications on using the rover vehicle type. If you guys agree I guess it make sense to make it standard and UGV seems a better fit as these controllers could also apply to differential drive robots.